DistroLauncher/WSLInfo.cpp

```diff
@@ -81,7 +85,8 @@ namespace Helpers {
         // One indication that the distro has graphical support enabled.
         bool isX11UnixSocketMounted() {
             const std::array cmds = {
-                L"ls -l /tmp/.X11-unix",
+                L"ls -l /mnt/wslg",
```

Comment:
`/mnt` is a configurable target for mounting Windows drive. I think it’s the same for wslg (but need confirmation). I would thus not rely on that path.
However, we should check that /tmp/.X11-unix is not a dangling symlink.

Reply:
In that case, listing /mnt/wslg might be misleading. I'll do some tests just got the sake of learning, but I'll take that command out.

Reply:
You are absolutely right! If one changes the `[automount].root` path in `/etc/wsl.conf`, wslg gets mounted in the new path provided. The symlink `/tmp/.X11-unix` is adjusted as well.

Reply:
`ls -l /tmp/.X11-unix | grep wslg` doesn't prove the symlink is resolved. `ls -l /tmp/.X11-unix/` does.

Reply:
There are multiple ways to detect that the symlink we point to exists:
`readlink -e /tmp/.X11-unix` returns something (and exits 0) if the target exists, 1 otherwise without any output.

Jibel suggest also use -e in bash -c for instance. We can even find more creative ways for sure :)

Reply:
That also means we will have trouble to know when to stop thinking about that... 😅